regtests/ww3_tp2.17/input/ww3_shel.inp

```diff
@@ -54,8 +54,8 @@ $                               see w3iogo.ftn for additional doc.
 $          
 $          
 $
-   20151214 020000 3600 20151214 040000
-  N
+ 20151214 000000 150 20151214 040000
```

Comment:
@aliabdolali I thought you reverted these changes in time steps to their original values?

Reply:
We will revert them after our final checks, now we need more freq outputs for our tests

Reply:
Ali, i think we can revert now. Thanks Jessica ...